README.adoc

```diff
@@ -115,6 +124,25 @@ our daps2docker project: https://github.com/openSUSE/daps2docker
 2. Clone the daps2docker repository.
 3. Run  `./daps2docker.sh /PATH/TO/DOC-DIR` or `/daps2docker.sh /PATH/TO/DC-FILE`.
 
+Building upstream docs
```

Comment:
Given that these upstream guides are in another repo, I'd suggest not to document the procedure in this repo but to add a repo readme in the other repo.
Otherwise, the content of this readme will just become outdated when a developer changes anything and nobody from the developer team will notice. If you put these instructions in the other repo's readme, 

This repo readme should have a link to the other readme though.

Reply:
So, I agree and disagree at the same time. There is a README in the other repo (which, as we discussed, needs to be moved to the top-level) but I think it's a good thing these instructions are here. It's a consistent reference point to *all* documentation for cloud. A lot of people will look here for these instructions. The other repo is just a body, mostly?